lw/http/testing/test.cert

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
+MIIC5TCCAc2gAwIBAgIRAMyDqNALqORpspUzRuRIBcEwDQYJKoZIhvcNAQENBQAw
+ADAgFw0yMDExMTAwMzMxMTVaGA8yMTIwMTExMDA5MzExNVowADCCASIwDQYJKoZI
+hvcNAQEBBQADggEPADCCAQoCggEBAMaH0iDnJzRzUm8aoJcTl/0eIPzBPYa6D/Hs
+0k4buNsV2nBFkmjHwU0Mglr69VO3Yoix8xUQ+gHl7GCfPWVhcsCkZaipnhM9fZb0
+T+426xssta+SF2tNrglv6/tgGCalElCr8r5HzuNGECpb6uw9/G2I0hb1GehLM4Yt
+uhrVKv4T6wBTAlaAz/rMmEwguK+z/W6+Vgq0HQF3PMmpNuH6wr3fVrqjNt93Bgn4
+RO4qbX/6yg9l+4n9u/u51k62H4qutOt5+JvkvmT+/caxTZ7PlO2K2Hm7wQ5KRb09
+G2gqMcHAUAqp4knSgdLwtkELd/AfJRFHWLJqnYS2oG4Euz2LodsCAwEAAaNYMFYw
+IQYDVR0OBBoEGKwIKMI6iKRlwO7XaR3N0yMcKYU2KlnSYjAMBgNVHRMBAf8EAjAA
+MCMGA1UdIwQcMBqAGKwIKMI6iKRlwO7XaR3N0yMcKYU2KlnSYjANBgkqhkiG9w0B
+AQ0FAAOCAQEAEUX1LTDeTPHxR0aOSlmVbXzx7dPpMbPoR0YzjBZdfNuUXEga8KsB
+iAlxUEBaaigJBtt8SiR3j21YiLq8kBE3jdtrVrlQN64j3ixtjsjzKhLTAUX0hKTv
+Bc1GHWJF2DvomOguS9mNDOe7NX2aoZxL2XmBuTwI+0oP2mjBftWqzpWvwnTjyoJM
+0603s+gEZ1HfFpIaWtyYRrFRnyJB0RUSsRE1e7MzDUNJ8F0pZMydXAY55I2w7Zla
+aXzMZ/99HGaCDtfrQasaArVKqPA0MblB5EU3zSdvp8eGozCBAWELlM162L+0eX2f
+vFqFPPzuXHSjRQvYsftCyhcnkMv6WwKbsA==
+-----END CERTIFICATE-----
```

Comment:
This key and cert are for testing purposes only. This is documented in the path (`lw/http/testing`), the file's name (`test.cert`), and the build file which marks the data bundle for this as `testonly = True`.

That last part will make the build system refuse to compile anything that references this file which is not part of a test target. This means I can not create a binary distribution target which references this file, nor references any other build targets which reference this file.

The file/path naming convention also makes it clear in the code itself that these files are meant only for testing and should not be touched outside of that. If I write any code which is not a test, or testing related, and include a file with `testing` in the path, that is a clear indication that something is wrong.

Finally, this key/cert pair are a self-signed pair without an associated certificate authority. This means that no web browser or other real-world TLS-enabled client would accept it. So even if I circumvented everything above, these are useless in a real application because nothing would talk to it. In my tests I am not checking the certificate authority and thus don't have that problem.